workbench-core/example/infrastructure/README.md

```diff
@@ -60,8 +58,17 @@ rushx cdk:destroy -f
 ```bash
 ./scripts/cleanup.sh -u <USER_POOL_ID> -d <DYNAMO_DB_TABLE_NAME> -r <REGION> -p -c
 ```
+#### Get UserPool Id and DynamoDB Table
+```bash
+./scripts/cleanup.sh -r <REGION> -i -t -c
+```
+#### OR
+
 After `rushx cdk:deploy` you can find:
 
 1. USER_POOL_ID [here](./src/config/testEnv.json#L13)
 
-2. DYNAMO_DB_TABLE_NAME [here](./src/config/testEnv.json#L12)
\ No newline at end of file
+2. DYNAMO_DB_TABLE_NAME [here](./src/config/testEnv.json#L12)
+
+
+
```

Comment:
if youre in this file again this PR, you can delete these extra lines, but if not dont worry about it